install/docker/index.markdown

```diff
@@ -31,25 +31,46 @@ And you will likely need to log out and back into your user account for the chan
 
 ## Running MoveIt Containers
 
-To run a Debian-installed container of MoveIt with graphics support:
+To use the command line only (without GUI interfaces like Rviz) in a non-persistent container, run this command:
 
-    wget https://raw.githubusercontent.com/ros-planning/moveit/master/.docker/gui-docker -O gui-docker && chmod +x gui-docker
-    ./gui-docker -it --rm moveit/moveit:melodic-release /bin/bash
+    docker run -it moveit/moveit:master-source bash
 
-This will attempt to use nvidia-docker if hardware and drivers are available.
+To use Rviz (and other GUIs), you need to set up hardware acceleration for Docker as described [here](http://wiki.ros.org/docker/Tutorials/Hardware%20Acceleration#nvidia-docker2). First, install nvidia-docker2 according to [these instructions](https://github.com/nvidia/nvidia-docker/wiki/Installation-(version-2.0)).
 
-**Command-Line Only**
+Then, use the commands below to:  
```

Comment:
I think we can keep the old, simpler instructions, if we stick with gui-docker from https://github.com/ros-planning/moveit/pull/1532